packages/design-system/src/components/Pagination/Page.tsx

```diff
@@ -0,0 +1,50 @@
+import Button from '../Button/Button';
+import React from 'react';
+
+export interface PageProps {
+  /**
+   * Sets a custom url for links. Optional.
+   */
+  customUrl?: string;
+  /**
+   * Defines the page number.
+   */
+  index: number;
+  /**
+   * Renders current page if true, other links if false.
+   */
+  isActive: boolean;
+  /**
+   * A callback function used to handle state changes.
+   */
+  onPageChange?: (evt: React.MouseEvent) => void;
+}
+
+export default function Page({
+  customUrl,
+  index,
+  isActive,
+  onPageChange,
+}: PageProps): React.ReactElement {
+  return (
+    <li>
+      {isActive ? (
+        <span
+          className="ds-c-button ds-c-button--transparent ds-c-pagination__current-page"
+          aria-current="true"
+        >
+          {index}
+        </span>
+      ) : (
+        <Button
+          variation="transparent"
+          href={customUrl ? `${customUrl}/${index}` : `#${index}`}
```

Comment:
Due to some technical limitations, all the single-page apps in one of our child sites use hash routing, so this default url could be problematic. Do these have to be actual links with hrefs, or could they be buttons styled like links? @davidakennedy, are they better as links?

Reply:
My assumption is Pagination updates the URL, which causes the browser history stack to update, making link the correct option. I'm still digging into this

Reply:
Per DK
> In the case of the pagination component, I’d advocate for links over buttons. Because when you interact with these the route will change. You’re going to a place vs. performing an action.

So links it is!

Reply:
@pwolfert I just pushed a fix up for this href problem, though I'm not seeing it on our demo link. React code examples should have a prop for `renderHref` that takes a function.